examples/quickstart-sklearn-tabular/sklearnexample/client_app.py

```diff
@@ -0,0 +1,82 @@
+import argparse
+import warnings
+
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import log_loss
+from flwr.common import Context
+from flwr.client import NumPyClient, ClientApp
+
+from example.task import (
+    set_model_params,
+    get_model_parameters,
+    set_initial_params,
+    load_data,
+    UNIQUE_LABELS,
+)
+
+
+class FlowerClient(NumPyClient):
+    def __init__(self, model, X_train, y_train, X_test, y_test):
+        self.model = model
+        self.X_train = X_train
+        self.y_train = y_train
+        self.X_test = X_test
+        self.y_test = y_test
+        self.unique_labels = UNIQUE_LABELS
+
+    def get_parameters(self, config):
+        return get_model_parameters(self.model)
```

Comment:
We are trying to update the examples in such a way that this method is no longer needed. For this, we need to initialize the global model when constructing the `ServerApp`.

Reply:
Sorry, I don't quite get it. This function, either directly or not (so either using `self.get_parameters` or `get_model_parameters`), is needed since we need to get the model parameters each round, not only the global one once. Also, in the mlx example it exists.

Reply:
yes. we want to retain the functionality to extract the parameters (in this case the `get_model_parameters()` function) but having the `get_parameters()` method seems in many cases redundant since the only reason why it exists is so, if the strategy doesn't initialise global model, one random client can be sampled and call the `get_parameters()` method. But it seems to be better to always initialise the global model at the strategy so this `get_parameters()` is not needed (i.e. `fit()` can directly call the `get_model_parameters()` function when returning).